src/ui/hooks/recordings.ts

```diff
@@ -282,6 +282,7 @@ export function convertRecording(
     title: "title" in rec ? rec.title : undefined,
     duration: rec.duration,
     private: "private" in rec ? rec.private : undefined,
+    isProcessed: "isProcessed" in rec ? rec.isProcessed : undefined,
     isInitialized: "isInitialized" in rec ? rec.isInitialized : undefined,
```

Comment:
What's the diff between processed and initialized?

Reply:
Not actually sure — can investigate